README.md

```diff
@@ -50,24 +48,33 @@ This will ensure any dependencies are ready before launching Xcode.
 
 You can also open the project by double clicking on Simplenote.xcworkspace file, or launching Xcode and choose `File` > `Open` and browse to `Simplenote.xcworkspace`.
 
+Once you have opened Simpleonte iOS in Xcode, depending on your setup, you may need to make a few changes before you can build the app.  In Xcode hit `Command + B` and see if you get any errors.  
+
+If you see `The server SSH fingerprint failed to verify` before you can build Simplenote you will need to mark the app as trusted.  To do this, tap on the warning and hit Trust
+
+If the build fails with an `Authentication failed because the credentials were missing` error most likely you need to update the Swift Packages before being able to build the app. We use Swift Package Manager for some internal dependencies which can be found on Github.  To fetch these packages, connect Xcode to Github by going to Xcode > Preferences > Accounts, then enter your Github account details.  To be able to fetch these dependencies Xcode will need to be connected to a Github account via [SSH](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+
+Once you have trusted the app and the SPM packages are downloaded you should be able to build the app.  Try `Command + B` again and make sure that it builds correctly.
 
 ## Setup Credentials
 
-Simplenote is powered by the [Simperium Sync'ing protocol](https://www.simperium.com). We distribute **testing credentials** that help us authenticate your application, and verify that the API calls being made are valid. Once the Simperium account is created, you can register an app and access the APP ID and necessary API keys.
+Simplenote is powered by the [Simperium Sync'ing protocol](https://www.simperium.com). To be able to connect a development build of Simplenote iOS to the Simperium syncing service you will first need to setup app credentials.
+We distribute **testing credentials** that help us authenticate your application, and verify that the API calls being made are valid. 
 
 **⚠️ Please note → We're not accepting any new Simperium accounts at this time.**
+First you will need to setup an account with [Simperium.com](https://www.simperium.com) 
+
+After creating your account, login and click on `Add an App`, give the app a name, and you will be taken to your test apps details page.  From there you need to copy the `App ID` and the `Default API Key`
```

Comment:
I've reworded this section in the macOS app ([PR here](https://github.com/Automattic/simplenote-macos/pull/1035/files)).

I'd suggest dropping these two sections, since signup in simperium.com is not allowed (and we're not sure if it'll ever be, back again). WDYT?

Reply:
I had thought possibly the same thing.  But I think we should say `something` cause if a non a8c OSS found simplenote and wanted to contribute there would be no details on the readme about why they can't login.